roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/StreetSection.java

```diff
@@ -57,6 +60,16 @@ public void addCar(ICar car) {
         carQueue.addLast(car);
         carPositions.put(car, INITIAL_CAR_POSITION);
         this.carCounter++;
+
+        // call carDelivered events for last section, so the car position
+        // of the current car (that has just left the last section successfully)
+        // can be removed (saves memory)
```

Comment:
TODO?

Reply:
no, it's done below. this is just a comment why the lines below are necessary